api/types/transfer/imagestore.proto

```diff
@@ -37,6 +37,10 @@ message ImageStore {
 	// extra_references are used to set image names on imports of sub-images from the index
 	repeated ImageReference extra_references = 6;
 
+	// Content retention policy
+
+	ContentPolicy content_policy = 8;
```

Comment:
I see there's a 10th proto field below -- was there previously a different 8th field, and would introducing this cause any conflict there? What is the cause for the gap?